plugin/pkg/plant/admission.go

```diff
@@ -35,7 +35,7 @@ import (
 
 const (
 	// PluginName is the name of this admission plugin.
-	PluginName = "PlantValidator"
+	PluginName = "Handler"
```

Comment:
Hmm, why we rename it to `Handler`? Currently if someone wants to disable this admission plugin, he would need to pass `--disable-admission-plugins "Handler" \` which is not self-explaining, right? Also the respective plugin for Shoot is `ShootValidator`.

Reply:
Good point. The problem was that this admission plugin is in pkg/plant but does not have a dedicated purpose (like pkg/shoot/validator). It does several things at once. I will clean it up 👍

Reply:
Thank you!